sdk/spring/azure-spring-cloud-autoconfigure/src/main/java/com/azure/spring/autoconfigure/storage/resource/FileStorageResource.java

```diff
@@ -15,8 +15,8 @@
 import java.io.File;
 import java.io.FileNotFoundException;
 import java.io.IOException;
-import java.io.OutputStream;
 import java.io.InputStream;
+import java.io.OutputStream;
```

Comment:
I guess you also did a auto import cleanup here, usually I'd not suggest to do that in a single PR , since it just brings a lots of noise 

open a new PR just contains imports cleanup next time would better